assets/tests/testHelpers/mockHelpers.ts

```diff
@@ -92,3 +94,7 @@ export const mockTileUrls = (): void => {
     }
   })
 }
+
+export const mockScreenSize = (screenSize: DeviceType): void => {
+  ;(useScreenSize as jest.Mock).mockImplementation(() => screenSize)
```

Comment:
```suggestion
  jest.mocked(useScreenSize).mockReturnValue(screenSize)
```

Reply:
We have a lot of this `;(<thing> as jest.Mock)` syntax in our codebase, and it's a little unclear why that was chosen over say, `jest.mocked`, so as of recently, we've been trying to convert to using `jest.mocked`.

But this isn't documented anywhere, hmm.